### ISSUE TITLE
My NEO has been lost during the transaction from neotracker to ledger S

### ISSUE THREAD

**Issue by Dooctoore** [CityOfZion/neon-wallet#435]
Hello
This is happened when i decided to join Qlink ico crowdsale. I filled the KYC form however i just gave up to join crowdsale. Today i would like to check my NEO wallet and saw zero NEOs in my wallet. Thereafter i immediately check my neotracker adress and saw my NEOs there. Then i decided to secure my NEOs just simply sending to my Ledger S. I copy and check the ledger adress 3 times and sent the NEOs from neotracker. However in my ledger acount i cant see my NEOs. What can be the problem what do you think ? You can see my orders as attached. Please help me i am noob. Thank you in advance
![ekran resmi 2017-12-23 00 14 50](https://user-images.githubusercontent.com/34779849/34312590-581211ce-e776-11e7-8a30-ac10c8e9c023.png)

**Comment by Dooctoore** [comment]
I just realized that when i click my neo ledger s adress as you see in the attachments i can see my balance on neotracker but i  see zero balance when i open my ledger nano s

**Comment by coranos** [comment]
Is the block height zero on the neon-wallet?
It could be network problems.

**Comment by OrEagle97327** [comment]
I have the same problem.  We are not the only ones.  See link 42 for more discussion.

https://github.com/CityOfZion/neon-wallet/issues

**Comment by coranos** [comment]
This is a completely different issue than yours.

He sees his balance on neotracker, and not in the wallet. This is usually a firewall or network issue.

**Comment by grishatataru** [comment]
@coranos 
So i will try to explain what happened in my case:
I originally bought Neo on Bittrex maybe 1 year ago,then i moved all my NEO to NEON Wallet.I was able to login and claim gas with no issues
Then i found out about the Ledger Nano S from my friend and i bought it on amazon.com and i setup as new, with 24 words/seeds.
I moved all my NEO from NEON WALLET to the Ledger Nano S.
I was able to login into the NEON Wallet via ledger with no issues, claiming gas every month and my balance was always showing my NEOS.
One day i opened Neon Wallet as usual ,connected my ledger nano, entered the 8 digit pin and clicked login in using ledger and then i discovered that my balance shows 0 and some gas,like 0.007 something.
I thought it was a glitch and i should not be worried,few days past and nothing was showing up.
Then i start doing research and realized im not the only one
I did get in contact with Ledger Support-NOTHING
I contacted on Twitter Neo Tracker-NOTHING
All i get is" SORRY there is nothing we can DO

Nobody besides me have access to my 24 seeds/key,i thought i was hacked but i do have other crypto like BTC LTC BTG and they are still on the Ledger.

I tried to reset my Ledger,restored it using 24 keywords but still showing 0

They were moved from this address:

AVjDZ7VVz7foeqFmdW2R5okbjr1pTEmEBs

to

AcWyPgfSVovTnE5rG6w3KNMCcoipjuDBxP

screen shot 2018-07-09 at 3 25 13 pm

screen shot 2018-07-09 at 3 26 40 pm

Now when i click to receive NEO,gives me the OLD public address

screen shot 2018-07-09 at 3 27 48 pm

How can i access the New Adress?

I tried to restore the PIW using the SEEDS and i can only see the old Adress/Current One with private key
Private KEY its attached the the AVjDZ7VVz7foeqFmdW2R5okbjr1pTEmEBs address

This happened on 4/18/2018, 11:39:19 PM and the new address was created at the same time

screen shot 2018-07-09 at 3 30 53 pm

I think i covered everything

Ill be waiting for ANY reply

Thanks again
